torchvision/prototype/transforms/_augment.py

```diff
@@ -78,18 +83,49 @@ def _get_params(self, flat_inputs: List[Any]) -> Dict[str, Any]:
             if not (h < img_h and w < img_w):
                 continue
 
-            if self.value is None:
+            if value is None:
                 v = torch.empty([img_c, h, w], dtype=torch.float32).normal_()
             else:
-                v = torch.tensor(self.value)[:, None, None]
+                v = torch.tensor(value)[:, None, None]
 
-            i = torch.randint(0, img_h - h + 1, size=(1,)).item()
-            j = torch.randint(0, img_w - w + 1, size=(1,)).item()
+            i = int(torch.randint(0, img_h - h + 1, size=(1,)))
+            j = int(torch.randint(0, img_w - w + 1, size=(1,)))
             break
         else:
             i, j, h, w, v = 0, 0, img_h, img_w, None
 
-        return dict(i=i, j=j, h=h, w=w, v=v)
+        return i, j, h, w, v
+
+    def _get_params(self, flat_inputs: List[Any]) -> Dict[str, Any]:
+        img_c, img_h, img_w = query_chw(flat_inputs)
+        return dict(
+            zip(
+                "ijhwv",
+                self._get_params_internal(
+                    img_c,
+                    img_h,
+                    img_w,
+                    self.scale,
+                    self._log_ratio,
+                    self.value,  # type: ignore[arg-type]
+                ),
+            )
+        )
+
+    @staticmethod
+    def get_params(
+        image: torch.Tensor,
+        scale: Tuple[float, float],
+        ratio: Tuple[float, float],
+        value: Optional[List[float]] = None,
+    ) -> Tuple[int, int, int, int, torch.Tensor]:
+        img_c, img_h, img_w = get_dimensions(image)
```

Comment:
In v2, we use the `query_chw` idiom, since in general the transform should not be bound to images and we can extract this information from multiple types. Since we already have an image here, we can extract directly and pass that.